src/course-outline/card-header/TitleButton.jsx

```diff
@@ -39,7 +39,7 @@ const TitleButton = ({
         className="item-card-header__title-btn"
         onClick={onTitleClick}
       >
-        <Truncate lines={1} className={`${namePrefix}-card-title mb-0`}>{title}</Truncate>
+        <Truncate.Deprecated lines={1} className={`${namePrefix}-card-title mb-0`}>{title}</Truncate.Deprecated>
```

Comment:
[question]: The PWG decided to depricate the Truncate component, since in 2025 we already have quite good support for [line-clamp CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/line-clamp). Does it make sense to move the related logic with the Truncate component to CSS now?